### DISCUSSION TITLE
DisableWAUAutoUpdate not Applying

### DISCUSSION THREAD

**Discussion by m1lkman** [Romanitho/Winget-AutoUpdate#207] [Q&A]
How do we deploy these settings with the app when setting up to run as SYSTEM? From the scripts, it seems that settings are being written to the ARP reg key but when running as system these values are ignored in favor of the settings.json. Can the main install script be modified that when the install is run as system to write values like DisableWAUAutoUpdate to the settings.json?

**Comment by Romanitho** [comment]
Not sure to understand. Requesting my joker @KnifMelti

**Reply by KnifMelti** [reply]
https://i.kym-cdn.com/photos/images/original/002/349/165/9a0.jpeg

**Comment by m1lkman** [comment]
Sorry, I don't think I fully grasp how the settings are getting applied during installation. I have noticed that despite setting the `-DisableWAUAutoUpdate` flag during installation, Winget-AutoUpdate is still in fact updating. I will collect more info and report back. I will also update this discussion topic.

**Comment by m1lkman** [comment]
What I'm seeing is when I install Winget-AutoUpdate.ps1 as a Win32 app install during Autopilot the WAU_DisableAutoUpdate value is not getting written to the registry. Looks like WAU_DoNotRunOnMetered is also not getting created. I'm assuming it's some defaultuser0 issue. The command line I'm running is `powershell.exe -NoProfile -ExecutionPolicy Bypass -file "Winget-AutoUpdate-Install.ps1" -Silent -DoNotUpdate -DisableWAUAutoUpdate -UpdatesAtLogon  -UpdatesInterval Monthly`

This is what's getting written to the ARP area. Any ideas?
![image](https://user-images.githubusercontent.com/10157301/197658099-38e5734b-0232-4d15-ac63-001f90ba393e.png)

**Reply by KnifMelti** [reply]
#88 maybe...
%systemroot%\sysnative... is the recommended way

**Comment by m1lkman** [comment]
I used that as a reference. Here is my win32 properties.

![image](https://user-images.githubusercontent.com/10157301/197659498-1699fd4d-adc9-4172-b790-a286a5e55623.png)

**Comment by m1lkman** [comment]
It's a sysnative issue. My bad. I see the values in WOW6432Node. Sorry

**Comment by KnifMelti** [comment]
@Romanitho why not stop telling the community to install via PowerShell.exe and instead install/uninstalll via your CMD:s (with their own filled in parameters) in which you provide the correct sysnative path?

**Reply by Romanitho** [reply]
Because it will just move the pb to the cmd file. Sysnative is just used by x86 execution. (And if you just want to change a argument you need to package another intunwin etc etc. 
I will add a check in the ps1 itself and rerun the script itself with sysnative thing.
